quickwit-backward-compat/test-data/index-metadata/v0-1106d4a0f7fba17fa3b9dd95052e8e4b.json

```diff
@@ -0,0 +1,77 @@
+{
+  "checkpoint": {
+    "0000000000": "0000000042"
+  },
+  "create_timestamp": 1789,
+  "doc_mapping": {
+    "field_mappings": [
+      {
+        "fast": true,
+        "indexed": true,
+        "name": "tenant_id",
+        "stored": true,
+        "type": "u64"
+      },
+      {
+        "fast": true,
+        "indexed": true,
+        "name": "timestamp",
+        "stored": true,
+        "type": "i64"
+      },
+      {
+        "fast": false,
+        "name": "log_level",
+        "record": "basic",
+        "stored": true,
+        "tokenizer": "raw",
+        "type": "text"
+      },
+      {
+        "fast": false,
+        "name": "message",
+        "record": "position",
+        "stored": true,
+        "tokenizer": "default",
+        "type": "text"
+      }
+    ],
+    "store_source": true,
+    "tag_fields": [
+      "tenant_id",
+      "log_level"
+    ]
+  },
+  "index_id": "my-index",
+  "index_uri": "s3://quickwit-indexes/my-index",
+  "indexing_settings": {
+    "commit_timeout_secs": 301,
+    "demux_field": "tenant_id",
+    "merge_policy": {
+      "demux_factor": 7,
+      "max_merge_factor": 11,
+      "merge_factor": 9
+    },
+    "resources": {
+      "heap_size": 3,
+      "num_threads": 3
+    },
+    "sort_field": "timestamp",
```

Comment:
nitpick one remark on the sort field, we may want to offer several sorting fields `timestamp` and `tenant_id` for example.
and I liked having `sort_field` and `sort_order` in a nested struct.

Reply:
@fulmicoton, nested or not nested?

Also, I have yet to implement multiple sort fields.